includes/class-plugin.php

```diff
@@ -319,12 +318,6 @@ public function enqueue_admin_scripts() {
      * @return void
      */
     public function enqueue_redis_metrics() {
-        global $wp_object_cache;
-
-        if ( defined( 'WP_REDIS_DISABLE_METRICS' ) && WP_REDIS_DISABLE_METRICS ) {
-            return;
-        }
```

Comment:
Removing this mean we're always loading `apexcharts.min.js` on the dashboard, even when metrics are disabled.

Can we call `Metrics::is_active()` here?

Reply:
Good catch!
If #263 would be merged we could move the whole method into the metrics class.